src/azure-cli/azure/cli/command_modules/privatedns/tests/latest/test_privatedns_commands.py

```diff
@@ -895,5 +898,86 @@ def test_ListRecordSetsAcrossType_MultipleRecordSetsPresent_ExpectMultipleRecord
         self.assertTrue(all(recordset in createdRecordsets for recordset in returnedRecordsets))
 
 
+@live_only()
```

Comment:
Is it ok to add comment here? paste this issue. https://github.com/kevin1024/vcrpy/issues/516

Reply:
Added comment.